persist/boltdb_test.go

```diff
@@ -196,9 +202,8 @@ func TestOpenDatabase(t *testing.T) {
 // with the wrong filemode (< 0600), which should result in an os.ErrPermission
 // error.
 func TestErrPermissionOpenDatabase(t *testing.T) {
-	if testing.Short() {
-		t.SkipNow()
-	}
+	t.Skip("can't reproduce on Windows")
```

Comment:
actually, I think I know what's happening here. Bolt uses `os.OpenFile` with the `os.O_CREATE` flag, meaning it will create the file if it does not exist. But it does *not* use this flag:
```go
O_EXCL int = syscall.O_EXCL   // used with O_CREATE, file must not exist
```
My guess if that if you don't use `O_EXCL` on Windows, it will simply overwrite the badly-permissioned file with a properly-permissioned file. This would explain the other permissions test failure as well, since it's the same deal: we use `O_CREATE` when opening log files, but not `O_EXCL`.

I will try fixing the latter by adding `O_EXCL` to our `NewFileLogger` function. If it works, we can modify our `bolt.Open` function as well. (And probably open an issue about it on their repo.)